### DISCUSSION TITLE
Matplotlib Issue - Can't Import The Package - cannot import name 'rcParams' from 'matplotlib.axes'

### DISCUSSION THREAD

**Discussion by ELITSZORD** [teamalgoritma/community#506] [Q&A] [answered]
Dear Algorit.ma's team and fellows,

![eror matplotlib](https://user-images.githubusercontent.com/114390619/211696777-834db059-e085-4792-9a90-cb890e5a4f63.png)

I have an issue with this package when I import it. The problem is  **ImportError: cannot import name 'rcParams' from 'matplotlib.axes' (C:\Users\Faisal Adhisthana N\anaconda3\envs\ult\lib\site-packages\matplotlib\axes\__init__.py)** .

I have tried ranging from reinstalling the anaconda, upgrading/downgrading the library version, updating the Jupiter notebook version, deleting/creating a new environment, restarting/upgrading the kernel, and so on. Do any of you guys have other solutions?

Best Regards,

Faisal

**Comment by ranydwi** [comment]
Dear Pak Fasial,

Terima kasih untuk pertanyaan yang masuk,
Sebelumnya ingin menanyakan terlebih dahulu version dari matplotlib dan juga numpynya Pak Faisal. Selanjutnya silahkan untuk buka jupyter notebook dari awal kembali.

Silahkan untuk mencoba menjalankan `from pylab import rcParams` sebelum melakukan importing matplotlib.

Referensi: https://stackoverflow.com/questions/59936356/importerror-cannot-import-name-rcparams-from-matplotlib 

Semoga membantu Pak
Best regards.

**Reply by ELITSZORD** [reply]
Dear,

Terimakasih atas responsenya. Dan ternyata setelah saya coba tetep gak bisa. Untuk versi numpy nya 1.24.1 & matplotlibnya 3.6.2

![matplotlib - 1](https://user-images.githubusercontent.com/114390619/211711776-7d8c352a-515a-4e8d-9485-218d823f2a08.png)

![v1](https://user-images.githubusercontent.com/114390619/211713858-1bb2a4c2-ea0a-4c13-b39f-d9aa4296ccd0.png)


Atau apakah ini context nya mengenai clash import? Mungkin ada solusi lainnya?
Thanks.

Best Regards,

Faisal

**Comment by ranydwi** [comment, marked as answer]
Dear @ELITSZORD 

Berdasarkan solving di kelas, permasalahan Bapak terjawab dengan membuat environment baru menggunakan python 3.9. Karena permasalahan ini sudah selesai, izin saya mark as answer discussion ini.

Regards,
Rany

**Reply by ELITSZORD** [reply]
Dear @ranydwi 

Terimakasih untuk rekan-rekan team algoritma yang sudah membantu saya.

Faisal
